Workspace/src/ping_test/src/gps_tester.cpp

```diff
@@ -0,0 +1,38 @@
+#include "ping_test.h"
+#include "timer.h"
+#include <chrono>
+#include <condition_variable>
+#include <ctime>
+#include <iostream>
+#include <mutex>
+#include <ros/ros.h>
+#include <sensor_msgs/NavSatFix.h>
+#include <std_msgs/Bool.h>
+#include <stdlib.h>
+#include <thread>
+
+int main(int argc, char **argv) {
+  // Initializes ROS, and sets up a node
+  ros::init(argc, argv, "gps_tester");
+  ros::NodeHandle nh;
+
+  // Set up publishers and subscribers
+  ros::Publisher gpsPub =
+      nh.advertise<sensor_msgs::NavSatFix>(GPS_SENSOR_TOPIC, 1);
+
+  sensor_msgs::NavSatFix msg;
+
+  msg.header.stamp = ros::Time::now();
+
+  msg.longitude = 0;
+  msg.latitude = 0;
+  msg.altitude = 0;
+
+  ros::Rate rate(1);
+  while (ros::ok()) {
```

Comment:
this publishes on a loop, do we really need to do that to return home?

Reply:
This node is just a test case i made to see if it would work, it wouldnt be running for competition. I had it publish gps coordinates then i disconnected one of my machines from wifi to see if the client machine would start backtracking

Reply:
Ok